docs/recipes/accounts.md

```diff
@@ -38,5 +38,75 @@ the amount of data you intend to store in the account.
   </CodeGroupItem>
 </CodeGroup>
 
+## Program Derived Address
+
+[Prgoram dervied address (PDA)][3] is like a normal address but
+
+1. falling off ed25519 curve
+2. using program to sign instead of private key
+
+::: tip
+Although PDA is derived by a program id, it doesn't means the PDA is owned by the same program. (Take an example, you can initialize your PDA as a token account which is an account owned by token program)
+:::
+
+### Generate a PDA
+
+To calculate a PDA, you need
```

Comment:
Nit:
`To calculate a PDA, you need the following:`

Reply:
fixed at [here](https://github.com/solana-dev-adv/solana-cookbook/pull/37/commits/42d7446110f35c59017ead4f16f161062538a1d2)